layers/meta-resin-genericx86/recipes-kernel/linux/files/realsense_camera_formats_linux-yocto_4.4.patch

```diff
@@ -0,0 +1,169 @@
+From 4b7c35a5363d8d976143896951889dacc57d306b Mon Sep 17 00:00:00 2001
+From: Scott Ware <scott.r.ware@intel.com>
+Date: Mon, 27 Nov 2017 14:19:53 +0000
+Subject: [PATCH 1/4] Add support for RealSense camera formats
+
+Upstream-Status: Backport
```

Comment:
needs also signed-ff by you, Theodor, in the correct order.
See https://github.com/resin-os/resin-intel/blob/master/layers/meta-resin-genericx86/recipes-kernel/linux/files/0001-Add-support-for-Quectel-EC20-modem.patch for example

Reply:
Will add it, but I do not understand why it needs to signed off by me, I did not touch these patches.